src/ListOfContacts/Dto/ListOfContactListsHubSpotEntity.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Newtonsoft.Json;
+using Skarp.HubSpotClient.Core.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Skarp.HubSpotClient.ListOfContacts.Dto
+{
+    public class ListOfContactListsHubSpotEntity: IHubSpotEntity
+    {
+        [DataContract]
+        public class MetaData
+        {
+            public string processing { get; set; }

```

Comment:
**issue** the properties of these classes carrying `[DataContract]` should preferably carry `[DataMember]` attributes that declares the serialized version of the property.

Also, the name should be PascalCased to follow the standard.

An example:
```
[DataMember(Name="processing")]
public string Processing { get; set; }